en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md

```diff
@@ -294,6 +379,143 @@ Run any of the following CTL commands to get keys for the API/API Product.
+
+1.  Export the API Product using `export api-product` command from the development environment (dev).
+
+    ```bash
+    $ apictl export api-product -n PetsInfo -e dev
+    
+    Successfully exported API Product!
+    Find the exported API Product at /home/wso2user/.wso2apictl/exported/api-products/dev/PetsInfo_1.0.0.zip
+    ```
+
+2.  Extract the exported API Product Project.
+
+3.  Commit the project to the git repository.
+
+4.  Run `vcs status` command to see the available changes that needs to be deployed to the production environment.
+
+    !!! example
+        ```bash
+        $ apictl vcs status -e prod
+        Projects to Deploy (1)
+
+        API Products (1) ...
+        1: [save]		PetsInfo-1.0.0: (PetsInfo-1.0.0)
+        ```
+
+3.  Import the **PetsInfo** Product into the production environment by running the following sample command.
+
+    !!! example
+        ```bash
+        $ apictl vcs deploy -e prod
+        Deploying Projects (1)...
+
+        API Products (1) ...
+        1: PetsInfo-1.0.0: (PetsInfo-1.0.0)
+        Successfully imported API Product
+        ```
+
+    Now the tool will automatically detect the target environment and create the **PetsInfo** Product in the target environment.
+    
+    !!! important
+        For deploying an API Product using `vcs deploy` command: 
+        
+        -   It is mandatory to have your API Product projects in a GIT based version control system.
+        -   It is mandatory to have `api_product_params.yaml` file inside each API Product Project. This is created by default when you exported an API Product using `export api-product`. In `api_product_params.yaml`. The below configuration section is used to deploy the API Product.
+
+        ```bash
+        vcs:
+            import:
+                updateApiProduct: true
+                preserveProvider: true
+                importApis: true
+                updateApis: false
+        ```
+
+        You can change the above fields accordingly.
+
+        | Field           | Description                                                                                 |
+        |-----------------|---------------------------------------------------------------------------------------------|
+        | updateApiProduct| Used to specify whether to update the API Product if it already exists during the deployment|
+        | preserveProvider| Preserve existing provider of the API Product after importing it                            |
+        | importApis      | Import the dependant API(s) along with the API Product if the dependant API(s) are not available in the target environment|
+        | updateApis      | Update the dependant API(s) in the target environment                                       |
+
+    !!! tip
+        If you have multiple API Projects to promote through CI/CD, you can commit those API Product projects as well to same repository. The rest of the process will be the same as above.
```

Comment:
Multiple API product projects could be promoted through CI/CD by commiting them to the same repository.